src/plugins/intel_gpu/tests/unit/test_cases/select_gpu_test.cpp

```diff
@@ -71,6 +71,85 @@ TEST(select_gpu_f32, select_basic) {
     test_select_basic<float>(false);
 }
 
+template <typename T>
+void test_select_basic_5d(bool is_caching_test) {
```

Comment:
I think it would be better to simplify the prototype as `void test_select_basic_5d()` instead of `template <typename T>
void test_select_basic_5d(bool is_caching_test)` because test case does not use template or is_caching_test argument.

Reply:
Updated as you commented.